src/scraper.py

```diff
@@ -197,9 +198,10 @@ def extract_skill(self):
         self.job_data['skills'] = skill_list
 
 
-# jd = JobData(job_title="Software Engineer",
-#              job_location="Raleigh",
-#              distance=200,
-#              company="",
-#              number_jobs=10)
-# jd.scrape_data()
+if __name__ == '__main__':
```

Comment:
Better to keep this commented out. Otherwise, it will be called everytime we run the tests.

Reply:
Are we going to call scraper.py for our scraping routine or are we going to create a new driving function for cron job? Something to think about. I'll comment it for now.